File: 1-easy/problems/2682-find-the-losers-of-the-circular-game/solution.ts
function circularGameLosers(n: any, k: any): boolean | number | string | any {
  """
  :type n: int
  :type k: int
  :rtype: List[int]
  """
  lookup = [false]*n
  idx = 0
  for (i in Array.from({length: n}, (_, i) => i)) {
      if (lookup[idx]) {
          break
      lookup[idx] = true
      idx = (idx+(i+1)*k)%n
  return [i+1 for i in Array.from({length: n}, (_, i) => i) if not lookup[i]];
}

export { circularGameLosers };

File: 1-easy/problems/2682-find-the-losers-of-the-circular-game/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def circularGameLosers(self, n, k):
        """
        :type n: int
        :type k: int
        :rtype: List[int]
        """
        lookup = [False]*n
        idx = 0
        for i in range(n):
            if lookup[idx]:
                break
            lookup[idx] = True
            idx = (idx+(i+1)*k)%n
        return [i+1 for i in range(n) if not lookup[i]]
